src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -200,6 +201,8 @@ def load_arguments(self, _):
         c.argument('uptime_sla', action='store_true')
         c.argument('fqdn_subdomain')
         c.argument('api_server_authorized_ip_ranges', validator=validate_ip_ranges)
+        c.argument('enable_apiserver_vnet_integration', action='store_true', is_preview=True)
+        c.argument('apiserver_subnet_id', validator=validate_apiserver_subnet_id, is_preview=True)
```

Comment:
It would be nice if this could be placed at the end of the parameter declaration section of `aks create` (align with `aks update`).

Reply:
fixed